directory.json

```diff
@@ -2248,7 +2248,10 @@
     "Function Scan": {
       "shortDescription": "Scans the result of a set-returning function (like unnest or regexp_split_to_table).",
       "importantFields": [
-        "Function Name"
+        "Function Name",
+        "Function Call",
+        "Filter",
+        "Rows Removed by Filter"
```

Comment:
Looks like this change was missed in https://github.com/pganalyze/pganalyze-docs/pull/319
(need to run gatsby locally to re-generate this file)